lib/scenic/adapters/postgres.rb

```diff
@@ -67,6 +79,23 @@ def create_materialized_view(name, sql_definition)
         execute "CREATE MATERIALIZED VIEW #{name} AS #{sql_definition};"
       end
 
+      # Updates a materialized view in the database.
+      #
+      # Attempts to maintain all applicable indexes on the materialized view
+      # after the view is recreated.
```

Comment:
I'd mention the raise:

```
@raise [MaterializedViewsNotSupportedError] if Postgres version is less than 9.3, which introduced materialized views.
```

Incidentally, it looks like if we really wanted to we could support this in older versions with triggers. https://wiki.postgresql.org/wiki/Materialized_Views